hack/make-helpers/make-build-all.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2023 VMware, Inc.
```

Comment:
Update copyright header.

Reply:
Updated copyright information as part of commit 2383ff5cbde826a1709eb415759868a4fae29529